app/src/main/res/values/strings.xml

```diff
@@ -11,11 +11,19 @@
     <string name="price">Price</string>
     <string name="display_price">$ %.2f</string>
     <string name="item_image_description">Item Image</string>
-    <string name="search_hint">Search...</string>
+    <string name="search_hint" tools:ignore="TypographyEllipsis">Search...</string>
```

Comment:
Maybe it's better to use the ellipsis character instead of ignoring the warning message.